ansible/os-devspace.yml

```diff
@@ -15,7 +15,7 @@
   vars:
 
     vm_image: "CentOS 7"
-    vm_flavour: m2.large
+    vm_flavour_ram: 24576
```

Comment:
a nitpicky suggestion, use 24000 in case people use units of 1000 instead of 1024 (since this is a minimum requirement)